doc/manual/parallel-computing.rst

```diff
@@ -492,6 +492,46 @@ and size 10. The channel exists on worker ``pid``\ .
 Methods ``put!``\ , ``take!``\ , ``fetch``\ , ``isready`` and ``wait`` on a ``RemoteChannel`` are proxied onto
 the backing store on the remote process.
 
+As an example, we can construct a ``RemoteChannel`` to a worker, ``put!`` an index inside, then ``take!`` it out:
+
+.. doctest::
```

Comment:
I think the doctest format is a little pickier than this, most of them look for `julia> ` on the inputs then non-prefixed output like you'd get at the repl